internal/pkg/describe/env.go

```diff
@@ -0,0 +1,104 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package describe
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/archer"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/aws/session"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/store"
+	"github.com/aws/aws-sdk-go/service/cloudformation"
+)
+
+type EnvironmentSummary struct {
+	Name         string `json:"name"`
+	AccountID    string `json:"accountID"`
+	Region       string `json:"region"`
+	IsProduction bool   `json:"production"`
+}
+
+type Environment struct {
+	*EnvironmentSummary `json:"environment"`
+	Applications        []*Application    `json:"applications"`
+	Tags                map[string]string `json:"tags,omitempty"`
+}
+
+// EnvDescriber retrieves information about an environment.
+type EnvDescriber struct {
+	env  *archer.Environment
+	apps []*archer.Application
+
+	store          envGetter
```

Comment:
Can we call this `envGetter` or something? It's not really the full `store` struct.